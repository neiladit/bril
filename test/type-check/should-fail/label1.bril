#label not present in code
main{

    v0: int = const 1;
    v1: int = const 3;
    v3: int = add v1 v2;
    jmp itsatrap;
    v4: int = const 4;
    print v4;
    check: bool = eq v3 v4;
    print check;
    
}
