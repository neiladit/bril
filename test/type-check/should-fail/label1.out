line 5: jump to undefined label itsatrap
jmp op error!
